granulate_utils/metrics/spark.py

```diff
@@ -268,7 +283,12 @@ def _spark_executor_metrics(self, running_apps: Dict[str, Tuple[str, str]]) -> I
         for app_id, (app_name, tracking_url) in running_apps.items():
             try:
                 base_url = get_request_url(self.master_address, tracking_url)
-                executors = rest_request_to_json(base_url, SPARK_APPS_PATH, app_id, "executors")
+                executors = rest_request_to_json(
+                    base_url,
+                    SPARK_APPS_PATH,
+                    app_id,
+                    "executors",
+                )
```

Comment:
revert this

Reply:
This is the git pre-commit hook, why revert? It's just alignment

Reply:
because it looks worse than the alternative

Reply:
@d3dave agree but it's the autoformatter, I'm not blocking @guybortnikov on this, we can fix the format later.